docs/system-overview.md

```diff
@@ -11,59 +11,80 @@ A separate component - the simulator - will handle all control of the simulation
+Only 1 car at the time will be able to run on this AS.
+The official can select which team, and thus which AS, is currently selected.
+When the team changes, not only the AS but also the car inside the virtual world will change.
 The sensor suite (sensor types and locations, defined in a custom `settings.json` file) and the car livery (looks of the car) are updated.
-The simulator keeps track of these details and passes them along to the virtual world to ensure accurate representation.
+The operator keeps track of these details and passes them along to the virtual world to ensure accurate representation.
 
-When a start signal is received, the simulator informs the current autonomous system of the mission and starts sending sensor data its way.
-When a stop signal is received, the simulator stops interacting with the attached autonomous system and informs the virtual world that the car must stop driving
+When the operator wants to connect an AS to the simulated world, it launches the ros bridge.
+Read more about the ros bridge below.
+If the operator wants to disconnect the AS from the simulated world it stops the bridge node and the connections are stopped.
 
-During a mission, the simulator keeps polling the world for 'referee state'. 
+During a mission, the operator keeps polling the world for 'referee state'.
 This is information that in a physical world would be relevant to the referee.
 Currently, this includes a list of down or out cones and the timestamp of when they went down or out, a list of lap times and a list of when the car went off-track.
 More information will for sure be added.
 
 What happens inside the simulation is stored in a single logbook.
-This includes all referee updates, which autonomous systems were selected and which tracks were used.
+This includes all referee updates, which ASs were selected and which tracks were used.
 If something unexpected went wrong like a system's crash or error, a short description of what happened is included in this logbook.
 It gives a timeline of everything that happened to always go back afterwards and check what happened.
 The logbook is stored on disk so that in the event of a whole system crash we will still have the logbook.
 It is also shown within the referee's web interface.
 
+## The ROS Bridge
+The ros bridge node connects to the simulated world inside Unreal Engine using AirSim (more on that later).
+On the one hand, it requests sensor data and passes it along on ROS topics to the current AS.
+On the other hand, it receives car control commands from the AS and forwards it to the virtual world.
+So it acts as a bridge between the two.
+
+The node that is launched pointing at the AS's ROS master so that it can publish and subscribe to topics within the AS.
+Phisically this node runs on the server where the Unreal world is being simulated.
+The node is launched by the Operator.
+
+When the operator launches the ros bridge it passes along some mission variables.
```

Comment:
inconsistent capitalization of operator. Maybe this happens in more places, so I would do a find and replace